ros/src/computing/planning/motion/packages/waypoint_follower/nodes/mpc_follower/vehicle_model/vehicle_model_bicycle_kinematics.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mpc_follower/vehicle_model/vehicle_model_bicycle_kinematics.h"
```

Comment:
Can you add a reference to where the kinematics and dynamics model comes from?

Reply:
I added reference in README.